Scripts/Data/Type/Transformation/FloatAdder.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace VRTK.Core.Data.Type.Transformation
+{
+    using UnityEngine.Events;
+    using System;
+    using System.Linq;
+
+    /// <summary>
+    /// Processes a given <see cref="float"/> and adds it to all other values in the collection and outputs the sum of all values.
+    /// </summary>
+    /// <example>
+    /// 1f + 2f + 3f = 6f
+    /// </example>
+    public class FloatAdder : CollectionAggregator<float, float, FloatAdder.UnityEvent>
```

Comment:
```suggestion
    public class FloatsAdder : CollectionAggregator<float, float, FloatAdder.UnityEvent>
```
To emphasize that it acts on multiple ones.

Reply:
Adder implies it's working on multiple as you need at least 2 numbers to perform an add operation :)